app/assets/stylesheets/base/fonts.css.scss

```diff
@@ -1,174 +1,171 @@
-}
-
-@font-face {
-  font-family: 'Avenir LT Std';
-  font-style: italic;
-  font-weight: 95;
-  src: local('Avenir LT Std 95 Black Oblique'), url('/fonts/AvenirLTStd-BlackOblique.woff') format('woff');
-}
-
-
-
-/* AVENIR NEXT CONDENSED */
-@font-face {
-  font-family: 'Avenir Next Condensed';
-  font-style: normal;
-  font-weight: 200;
-  src: local('AvenirNextCondensed-UltraLight'), url('/fonts/AvenirNextCondensed-UltraLight.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed';
-  font-style: italic;
-  font-weight: 200;
-  src: local('AvenirNextCondensed-UltraLightItalic'), url('/fonts/AvenirNextCondensed-UltraLightItalic.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed';
-  font-style: normal;
-  font-weight: 400;
-  src: local('AvenirNextCondensed-Regular'), url('/fonts/AvenirNextCondensed-Regular.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed';
-  font-style: italic;
-  font-weight: 400;
-  src: local('AvenirNextCondensed-Italic'), url('/fonts/AvenirNextCondensed-Italic.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed Medium';
-  font-style: normal;
-  font-weight: 500;
-  src: local('AvenirNextCondensed-Medium'), url('/fonts/AvenirNextCondensed-Medium.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed Medium';
-  font-style: italic;
-  font-weight: 500;
-  src: local('AvenirNextCondensed-MediumItalic'), url('/fonts/AvenirNextCondensed-MediumItalic.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed Demi Bold';
-  font-style: normal;
-  font-weight: 600;
-  src: local('AvenirNextCondensed-DemiBold'), url('/fonts/AvenirNextCondensed-DemiBold.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed Demi Bold';
-  font-style: italic;
-  font-weight: 600;
-  src: local('AvenirNextCondensed-DemiBoldItalic'), url('/fonts/AvenirNextCondensed-DemiBoldItalic.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed Bold';
-  font-style: normal;
-  font-weight: 700;
-  src: local('AvenirNextCondensed-Bold'), url('/fonts/AvenirNextCondensed-Bold.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed Bold';
-  font-style: italic;
-  font-weight: 700;
-  src: local('AvenirNextCondensed-BoldItalic'), url('/fonts/AvenirNextCondensed-BoldItalic.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed Heavy';
-  font-style: normal;
-  font-weight: 900;
-  src: local('AvenirNextCondensed-Heavy'), url('/fonts/AvenirNextCondensed-Heavy.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Avenir Next Condensed Heavy';
-  font-style: italic;
-  font-weight: 900;
-  src: local('AvenirNextCondensed-HeavyItalic'), url('/fonts/AvenirNextCondensed-HeavyItalic.woff') format('woff');
-}
\ No newline at end of file
+// /* #### Generated By: http://www.cufonfonts.com #### */
+
```

Comment:
I've commented it out for now.... we'll decide later what to do with it but it changes font styles a lot...